chain/x/zoracle/internal/types/events.go

```diff
@@ -2,8 +2,14 @@ package types
 
 // Event types
 const (
-	EventTypeRequest      = "request"
-	EventTypeReport       = "report"
+	EventTypeCreateDataSource   = "create_data_source"
+	EventTypeEditDataSource     = "edit_data_source"
+	EventTypeCreateOracleScript = "create_oracle_script"
+	EventTypeEditOracleScript   = "edit_oracle_script"
+	EventTypeRequest            = "request"
+	EventTypeReport             = "report"
+
+	AttributeKeyID        = "id"
 	AttributeKeyRequestID = "id"
```

Comment:
Remove this?

Reply:
No it shouldn't be removed because it's also use in Report event. I rename the value to  "request_id" though for clarity.